api/full.Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM rust:alpine3.11
 WORKDIR /usr/src/code-library-api
-COPY . .
+COPY api .
```

Comment:
`Dockerfile`s are each in the respective folder, so there's no `api` folder in `api` folder, the previous version should be the correct one isn't it?

Reply:
It depends from where you initiate the build process. If you start from root directory you have to specify it. Otherwise, it would be like the previous version. I don't mind, we just need to decide for one version.

Reply:
I mean, you put them each in the respective folder in this PR and it's not gonna work with the latest change

Reply:
What I meant is, it depends on the working directory. When you execute `docker build -f api/slim.Dockerfile .` from the project root, i.e. `code-library-backend`, then it works. If you execute it from the `api` folder then you need a `COPY . .` in the Dockerfile. The question is therefore, how would you execute it, from the root or microservice directory?

Reply:
:smiley: maybe I missing something, but in the dockerfile itself the following line `copy . .` implies copying from the current directory of the host to the current directory of the container. if you say `copy api .` it would look for `api` folder in the current directory of the host, for this to work we need to place dockerfile in the directory with `api` folder because you specify the build context with `.` in the end. `docker build -f api/slim.Dockerfile .` with this command you specify the correct dockerfile, but the dockerfile instcructions will fail because the file is located in the `api` folder and not int the bulid context

Reply:
so basically, it boils down to: either change the location of the file in the directory of change the copy command

Reply:
on circleci it doesn't really matter where the file is, as I can specify the build command the same way

Reply:
I just changed to path back to the old value.

Reply:
:+1: